### PR TITLE
Add aggregation support

### PR DESCRIPTION
**Pull request by J2thatsme**
Added Boolean values to occupancy datapoints for aggregation on akenza

### CODE REVIEW

File: types/elsys/ems-desk/uplink.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ function DecodeElsysPayload(data) {
         break;
       case TYPE_OCCUPANCY: // Body occupancy
         obj.occupancy = data[i + 1];
+        obj.occupied = !!obj.occupancy;
```
Comment (mariusgiger) [inline types/elsys/ems-desk/uplink.js:131]: this should be true if it is 1 or 2

File: types/elsys/ers-desk/uplink.js [hunk 2]
```diff
@@ -128,6 +128,7 @@ function DecodeElsysPayload(data) {
         break;
       case TYPE_OCCUPANCY: // Body occupancy
         obj.occupancy = data[i + 1];
+        obj.occupied = !!obj.occupancy;
```
Comment (mariusgiger) [inline types/elsys/ers-desk/uplink.js:131]: this should be true if it is 1 or 2

**Review by mariusgiger — COMMENTED** [review]
looks good, please check the mapping for Elsys because of the multiple states

File: types/elsys/ems-desk/uplink.js (hunk 1 above)
Comment (J2thatsme) [reply types/elsys/ems-desk/uplink.js:131]: Pretty sure that !!1 || !!2 equals true

File: types/elsys/ers-desk/uplink.js (hunk 2 above)
Comment (J2thatsme) [reply types/elsys/ers-desk/uplink.js:131]: same as above

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**